python/GangaLHCb/Lib/Applications/BenderBox.py

```diff
@@ -0,0 +1,520 @@
+# -*- coding: utf-8 -*-
+# =============================================================================
+## @file BenderBox.py
```

Comment:
General comment probably to the other  @ganga-devs/core or @egede  (I only put it at this line as this was the first instance). There are many PEP8 violations in this PR. How strictly are we requiring PEP8 compliance these days?